### PR TITLE
Updatedash

### PR DESCRIPTION
**Pull request by GAUTAMRAJU15**
it contains mutiline graph

### CODE REVIEW

File: src/components/Dashboard/Dashboard.js [hunk 1]
```diff
@@ -87,6 +89,7 @@ class Dashboard extends Component {
         dataSet.push(tempData);
         tempData = Object.assign({}, {});
       });
+      console.log(dataSet,'<<<<<<<dataSet>>>>>');
```
Comment (ShankyRana) [inline src/components/Dashboard/Dashboard.js]: remove

File: src/components/Dashboard/Dashboard.js [hunk 2]
```diff
@@ -74,11 +74,13 @@ class Dashboard extends Component {
...
         let user = campaign.uniqueUsers;
         let tempData = Object.assign({}, dataContent);
-
+        delete tempData['data'];
+        tempData['data']=[0, 0, 0, 0, 0, 0, 0];
         if(user && user.aggregations && user.aggregations.users.buckets.length) {
           user.aggregations.users.buckets.map(bucket => {
             tempData['label'] = campaign.campaignName;
             tempData['data'][Moment(bucket.key_as_string).day()] = bucket.visitors.sum_other_doc_count + bucket.visitors.buckets.length;
+            // console.log(tempData,'<<<<<TEMPDATA>>>>>>>>>>>>');
```
Comment (ShankyRana) [inline src/components/Dashboard/Dashboard.js]: remove

File: src/components/Dashboard/Dashboard.js [hunk 3]
```diff
@@ -80,7 +80,11 @@ class Dashboard extends Component {
           user.aggregations.users.buckets.map(bucket => {
             tempData['label'] = campaign.campaignName;
             tempData['data'][Moment(bucket.key_as_string).day()] = bucket.visitors.sum_other_doc_count + bucket.visitors.buckets.length;
-            // console.log(tempData,'<<<<<TEMPDATA>>>>>>>>>>>>');
+            var duck = '';
+            duck = Moment(bucket.key_as_string)._i;
+            duck = duck.split('-')[2];
+            duck = duck.split(1,2);
+            console.log(duck,'<<<<<Day>>>>>>>>>>>>');
```
Comment (ShankyRana) [inline src/components/Dashboard/Dashboard.js]: remove

File: src/components/InstallPixel/InstallPixel.js [hunk 4]
```diff
@@ -1,5 +1,6 @@
 import React from 'react';
 import {Animated} from 'react-animated-css';
+// import { browserHistory } from 'react-router';
```
Comment (ShankyRana) [inline src/components/InstallPixel/InstallPixel.js]: remove

File: src/components/InstallPixel/InstallPixel.js [hunk 5]
```diff
@@ -10,7 +11,9 @@ const InstallPixel = ({
   verifyPixelStatus,
   handlePixelCopy,
   toggleWebhook,
-  displayWebhookIntegration
+  displayWebhookIntegration,
+// toggleTag,
```
Comment (ShankyRana) [inline src/components/InstallPixel/InstallPixel.js]: remove

File: src/components/InstallPixel/InstallPixel.scss [hunk 6]
```diff
@@ -60,6 +60,19 @@
       font-size: large;
     }
   }
+  .tagmanager {
+    // border-radius: 5px !important;
```
Comment (ShankyRana) [inline src/components/InstallPixel/InstallPixel.scss]: remove

File: src/components/InstallPixel/InstallPixel.scss [hunk 7]
```diff
@@ -60,6 +60,19 @@
...
+    // position: relative;
+  }
+  .modal-tag {
+    // z-index: 100 !important;
```
Comment (ShankyRana) [inline src/components/InstallPixel/InstallPixel.scss]: remove

File: src/__test__/Campaign.test.js [hunk 8]
```diff
@@ -0,0 +1,54 @@
...
+  it('+++ contains input', () => {
+    let wrapper = shallow(<Campaign  store={store}/>);
+    expect(wrapper.find('input').length).toBe(3);
+  });
+  it('+++ contains Form', () => {
+    let wrapper = shallow(<Campaign  store={store}/>);
+    expect(wrapper.find('form').length).toBe(1);
+  });
+
+  // it('should respond to change event and change the state of the Campaign Component', () => {
```
Comment (ShankyRana) [inline src/__test__/Campaign.test.js]: remove

**Comment by GAUTAMRAJU15** [comment]
pls merge

**Pull request merged**